### PR TITLE
Enable syntax coloring in all examples

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] prettyPrint function is not called on developer.tizen.org
          because only part between 'body' tags is copied

[Solution] Apply changes from commit 4cb5dce14589f3ee5ac9df5c21e481884c5ded6c to all files.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/618

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Button.htm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<link href="../snippet.css" rel="StyleSheet" type="text/css">
 	<title>Button</title>
 </head>
-<body onload="prettyPrint()" id="content">
+<bodyid="content">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Button.htm]: \<body id="content"\>

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	<link href="../snippet.css" rel="StyleSheet" type="text/css">
 	<title>Mobile UI Components</title>
 </head>
-<body onload="prettyPrint()" marginwidth="0" marginheight="0" id="content">
+<body marginwidth="0" marginheight="0" id="content">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm]: Is it ok that "marginwidth="0" or "marginheight="0" " left?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Button.htm (hunk 1 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Button.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm (hunk 2 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm]: @dorlinski 
These properties do not matter for document, but should be removed.

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm [hunk 3]
```diff
@@ -7,7 +7,7 @@
 	<link href="../snippet.css" rel="StyleSheet" type="text/css">
 	<title>Mobile UI Components</title>
 </head>
-<body onload="prettyPrint()" marginwidth="0" marginheight="0" id="content">
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm]: @dorlinski 
Could you remove unnecessary attributes `marginwidth="0" marginheight="0"` ? Thank you.

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_component_list.htm]: Fixed

**Comment by mijin85cho** [comment]
Thanks for the update & review

**Pull request merged**